src/libraries/System.Net.WebSockets.Client/tests/ConnectTest.Http2.cs

```diff
@@ -40,13 +50,15 @@ public async Task ConnectAsync_VersionNotSupported_Throws()
             async server =>
             {
                 Http2LoopbackConnection connection = await server.EstablishConnectionAsync(new SettingsEntry { SettingId = SettingId.EnableConnect, Value = 0 });
-            }, new Http2Options() { WebSocketEndpoint = true }
+            }, new Http2Options() { WebSocketEndpoint = true, UseSsl = false }
```

Comment:
Why SSL is removed? Should both options (with and without SSL) be tested?

Reply:
Put it back